docs/UserGuide.md

```diff
@@ -181,9 +196,12 @@ If your changes to the data file make its format invalid, FoodWhere will discard
 | **Exit**  | `exit`                                                                                                  |
 | **Help**  | `help`                                                                                                  |
 | **Clear** | `clear`                                                                                                 |
-| **rAdd**  | `radd s/STALL_INDEX d/DATE c/CONTENT [t/TAGS]`<br> e.g., `radd s/3 d/2022-09-20 c/Great food!`          |
+| **rAdd**  | `radd s/STALL_INDEX d/DATE c/CONTENT r/RATING [t/TAGS]`<br> e.g., `radd s/3 d/2022-09-20 c/Great food!` |
```

Comment:
Perhaps you can add the `...` for `[t/TAGS]…​` as items with …​ after them can be used multiple times including zero times.